install_config/web_console_customization.adoc

```diff
@@ -112,7 +109,38 @@ window.OPENSHIFT_EXTENSION_PROPERTIES = {
   key2: "value2",
 }
 ----
-====
+
+[[extension-option-for-external-logging-solutions]]
+=== Extension Option for External Logging Solutions
+
+As of {product-title} 3.6, there is an extension option to link to external
+logging solutions instead of using {product-title}'s EFK logging stack:
+
+----
+'use strict';
+angular.module("mylinkextensions", ['openshiftConsole'])
+       .run(function(extensionRegistry) {
+          extensionRegistry.add('log-links', _.spread(function(resource, options) {
+            return {
+              type: 'dom',
+              node: '<span><a href="https://extension-point.example.com">' + resource.metadata.name + '</a><span class="action-divider">|</span></span>'
+            };
+          }));
+       });
+hawtioPluginLoader.addModule("mylinkextensions");
+----
+
+The URL to the logging stack you are wanting to access
+(`\https://extension-point.example.com`, in this example) must be a valid HTTPS
+URL.
+
+The `resource` argument is the object that the logs are for (for example, a pod,
+deployment configuration, or build).
+
+The `options` argument depends on the kind of object, but will contain
+information like the specific container to pull logs for (`options.container`),
+or which replication controller the logs are for when getting logs for a
+deployment configuration (`options.version`).
```

Comment:
The `options` object are the log options we pass to the log REST API endpoint. You've documented the two important options:

For pods, `options.container` specifies the pod container name (e.g., `"my-container"`)
For deployment configs, `options.version` specified the deployment version number (e.g., `"2"`)